### DISCUSSION TITLE
Question mark / exclamation point in waybar plus slight crashes

### DISCUSSION THREAD

**Discussion by chahidanas** [JaKooLit/Fedora-Hyprland#143] [Q&A] [answered]
I just noticed that a question mark or an exclamation point appeared in waybar, I don't know what does it mean but since it appeared the rainbow animation and waybar been buggy and I needed to reload the waybar each time for it to start working before breaking again

- I tried rebooting (directly to hyprland and starting with gnome before switching back to hyprland)
- Login out and in again

![Before-clicking](https://github.com/user-attachments/assets/a93042a0-52c7-4765-981b-b9598b0df5ad)
-Before clicklin on it
![After-clicking](https://github.com/user-attachments/assets/7d0cc310-8b75-47d4-9727-231ee2157751)
-After clicking on it (showing ":")

Does anyone know what does it mean or how to fix it?

**Comment by chahidanas** [comment]
Right after submitting my comment I found out it was the weather part that broke, somehow spamming SUPER+ALT+R fixed it.

This is not an actual fix so if you have information about it (even a link to the particular wiki section talking about it) please let me know.

**Reply by JaKooLit** [reply, marked as answer]
of course. sometimes it cannot communicate to weather server

https://github.com/JaKooLit/Hyprland-Dots/wiki/TIPS#%EF%B8%8F-weather-app-related-for-waybar-and-hyprlock

switch to python based if you want
